apps/nfw-react/src/app/project/cmp/Menu/Menu.cmp.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { Nav } from '../Nav';
+import { PizzaList } from '../PizzaList';
+import { SauceList } from '../SauceList';
+
+export const Menu = ({ props }) => {
+  return (
+    <>
+      <Nav />
+      <div className="menu">
+        <h1 className="menu__heading menu__heading--pizza">Pizza</h1>
+        <PizzaList props={props} />
```

Comment:
(must) Again props - looks like U're passing in hierarchy from root to leafs everything (coding nightmare :D)